docs/tutorials/getting_data.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
<p>I'm not sure we want to advertise this method. It reads the point_map from disk, which only works when the catalog has just been loaded. Could also be a bit confusing with the difference between hipscat and lsdb packages, and that we have to use the hc_structure.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/400/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>